DependencyInjection/Configuration.php

```diff
@@ -534,7 +534,7 @@ public function getConfigTreeBuilder(): TreeBuilder
                 ->scalarNode('max_level')->defaultValue('EMERGENCY')->end() // filter
                 ->scalarNode('buffer_size')->defaultValue(0)->end() // fingers_crossed and buffer
                 ->booleanNode('flush_on_overflow')->defaultFalse()->end() // buffer
-                ->scalarNode('handler')->end() // fingers_crossed and buffer
+                ->scalarNode('handler')->end() // fingers_crossed, buffer, filter, deduplication, sampling
```

Comment:
Note for a future improvement, the [`info()`](https://symfony.com/doc/current/components/config/definition.html#documenting-the-option) method can be used to make this details more accessible.